e2e/cypress/integration/channel/close_direct_group_spec.js

```diff
@@ -122,6 +122,21 @@ describe('Close group messages', () => {
         });
     });
 
+    it('MM-T474 - Through x button on channel sidebar item for favorited channel', () => {
```

Comment:
Please use the title that's in TM4J. Thanks Eli!:
```suggestion
    it('MM-T474 - GM: Favorite, and close', () => {
```

Reply:
👍